index.html

```diff
@@ -2,16 +2,23 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <link rel="stylesheet" href="style.css">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>first mobile app</title>
     </head>
 
     <body>
        
             <header class="main-header">
                 <a href="#" class="header-text">John Doe</a>
+                <div class="desktop-header"> 
+                    <div>Protofilio</div>
+                    <div> About</div>
+                    <div> Contact</div>
+                    <a href="mohisadeghi302@gmail.com">
+                        <img  class="envelope" src="/img/env.png" alt="email icon"></a>
+                </div>    
```

Comment:
- [ ] Dear @mohisa302. Although you add an image I can see you used a '/' to refer to the source. This will only work for a local repository. Please instead use `./` to refer to a relative path.
> EG.  `src="/img/env.png"` to `src="./img/env.png"`